doc/translations.md

```diff
@@ -7,16 +7,16 @@ Translation Process
 -------------------
 
 The strings used in the UI of Friendica are translated at [Transifex] [1] and then included in the git repository at github.
-If you want to help with translation for any language, be it correcting terms or translating friendica to a currently not supported language, please register an account at transifex.com and contact the friendica translation team there.
+If you want to help with translation for any language, be it correcting terms or translating friendica to a currently not supported language, please register an account at Transifex.com and contact the friendica translation team there.
```

Comment:
While I agree that the service name should be uppercase, I don't think that the URLs should be too. Could you please change `Transifex.com` and `www.Transifex.com` back to lowercase?

Reply:
Yes, sorry, I have been a little trigger-happy with the replacement.